capstone/src/main/java/com/google/sps/servlets/BlobstoreUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+
+public final class BlobstoreUtil {
```

Comment:
Can we add a test for this util class as well? It's good practice to have tests for all classes (except for Constants-like classes, etc).

Also, should we be moving this to the clubs repo since it has to do with Blobstore?

Reply:
Done! I created a utils folder for this file since it applies to multiple Blobstore servlets. Thank you!